src/components/lyne-link/lyne-link.stories.js

```diff
@@ -0,0 +1,185 @@
+import {
+  ColorCharcoalDefault,
+  ColorWhiteDefault
+} from 'lyne-design-tokens/dist/js/tokens.es6';
+import getMarkupForSvg from '../../global/helpers/get-markup-for-svg';
+import { h } from 'jsx-dom';
+import readme from './readme.md';
+
+const wrapperStyle = (context) => {
+
+  if (context.args.variant === 'positive') {
+    return `background-color: ${ColorWhiteDefault};`;
+  }
+
+  return `background-color: ${ColorCharcoalDefault};`;
+
+};
+
+const Template = (args) => (
+  <lyne-link {...args}>
+    <span slot='icon'>{getMarkupForSvg(args.icon)}</span>
```

Comment:
Mini-improvement: this way, the slot is still rendered if no icon is configured via attribute. If you change it to this, we get rid of the empty slot:
```
{args.icon &&
      <span slot='icon'>{getMarkupForSvg(args.icon)}</span>
    }
```